project/src/main/webapp/event-details.css

```diff
@@ -33,3 +33,17 @@
 .alert {
   display: none;
 }
+
```

Comment:
want also img size limit (sometimes the img is huge)
img {
width: xxxx
height: xxxx
}

Reply:
Discussed over chat - cropping and resizing don't really work out well here so we will stick with properly-sized images